src/components/Frame.css

```diff
@@ -44,3 +44,48 @@
   color: #e79600;
   border-color: #e79600;
 }
+
+.frame-container {
+  position: relative;
+  height: 300px;
+  width: 300px;
+}
+
+.top-left {
+  position: absolute;
+  top: -15px;
+  left: 0px;
+}
+
+.top-center {
+  position: absolute;
+  text-align: center;
+  top: -15px;
+
+  width: 100%;
+}
+
+.top-right {
```

Comment:
`.Frame-container-text .top-right`, `.Frame-container-text .bottom-left`, etc.

Combine all style properties that apply to `.Frame-container-text` -- ie, `position: absolute;`